var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -54,6 +54,8 @@ class Sz(CMakePackage):
     depends_on('hdf5', when="+hdf5")
     depends_on('netcdf-c', when="+netcdf")
 
+    patch('ctags-only-if-requested.patch')
```

Comment:
@eugeneswalker , this won't work either. Some of the earliest version of SZ use autotools instead of CMake, and your patch will fail to apply.   Please apply the when clause I suggested instead.

Reply:
Sorry! Missed that. Fixing now.